images/crossplane-aws/tests/install.sh

```diff
@@ -1,6 +1,27 @@
 #!/usr/bin/env bash
 
-set -o errexit -o nounset -o errtrace -o pipefail -x
+set -o errexit -o nounset -o errtrace -o pipefail
+
+# If the image is on cgr.dev and we have a cred helper for that available, use
+# it to generate creds we can pass to the Crossplane control plane.
+# This is necessary when pushing to a private repo, since the Crossplane
+# control plane needs to pull the image to get configuration from it.
+tmp=$(mktemp -d)
+echo "{}" > ${tmp}/config.json
+if [ -x "$(command -v docker-credential-cgr)" ]; then
+  token=$(echo "cgr.dev" | docker-credential-cgr get)
```

Comment:
Can you pass in the registry via an environment variable extracted from the `IMAGE_NAME`?

Reply:
I thought about it, I can do that in a followup when/if it's a problem.